gubernator/github/classifier.py

```diff
@@ -133,7 +126,8 @@ def get_labels(events):
     Returns:
         labels: the currently applied labels as {label_name: label_color}
     '''
-    labels = []
+    if labels is None:
```

Comment:
use implicit t/f

Reply:
Google Python style guide on ["Default Argument Values"](https://google.github.io/styleguide/pyguide.html?showone=Default_Argument_Values#Default_Argument_Values) does it this way.

PEP8: "Comparisons to singletons like None should always be done with is or is not, never the equality operators.

Also, beware of writing if x when you really mean if x is not None -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!"

Reply:
IMO this does not apply here, given that if it is `None` you set it to `[]`. There is no behavioral difference between `[]` and `None`, you're just trying to initialize the value to a mutable type safely.

The style guide is warning people against `def foo(whatever=[]):` because it is not obvious that every call to `foo()` uses the same list instance.

We are making no attempts to distinguish between `None, 0, '' and []` here. In fact if for some reason we call `get_labels(labels=())` we definitely want to replace this with `[]` because otherwise we blow up later when we attempt to call `append()` on a tuple.

Personally I like the `labels = labels or []` for this particular instance